packages/xml-views-validation/src/validators/elements/use-of-deprecated-class.ts

```diff
@@ -26,18 +26,14 @@ export function validateUseOfDeprecatedClass(
     const deprecatedInfo = ui5Class.deprecatedInfo;
     const issues: UseOfDeprecatedClassIssue[] = [];
 
-    const commonIssueFields = {
+    const deprecatedIssue: UseOfDeprecatedClassIssue = {
```

Comment:
Looks ok, but now you don't need the `as` workarounds

Reply:
Right, I've removed them